configfiles.rst

```diff
@@ -940,9 +943,26 @@ configured in ``/etc/ld.so.conf`` (libraries).
  capability.json
 *****************
 
+Native runtime / non-OCI-Mode
+=============================
+
+In {Singularity}'s default configuration, without ``--oci``, a container started
```

Comment:
I wonder if it's worth mentioning here that containers can run in OCI-mode even when the `--oci` flag is not passed (though, crucially, _not_ in "SingularityCE's default configuration", so what's written here is of course correct), just so people avoid viewing `--oci` and OCI-mode as an if-and-only-if situation...

Reply:
I'm not sure yet... I don't want to have all the similar pieces in the docs repeating the same thing.

Will scribble this note onto a post-it, and might be something to deal with once the 'new in 4.0' and OCI mode specific content is developed.